ibis/tests/all/test_window.py

```diff
@@ -15,6 +16,63 @@
 )
 
 
+def ntile(x, bucket: int):
+    """
+    NTILE divides given data set into a number of buckets.
+
+    It divides an ordered and grouped data set into a number of buckets
+    and assigns the appropriate bucket number to each row.
+
+    Return an integer ranging from 0 to `bucket - 1`, dividing the
+    partition as equally as possible.
+
+    Parameters
+    ----------
+    x : pandas.core.groupby.generic.SeriesGroupBy ||
```

Comment:
can you type x in the signature

use the nomeclature
```
x : pandas.core.generic.SeriesGroupBy, ....
```

Reply:
ok sounds good. thanks!

Reply:
I applied the changes in the function signature. not sure if I understood correctly .. let me know if I need to change anything else.